### PR TITLE
DEV: Add roughing pump subclass specific to the Ebara EV-A03-1

### PR DESCRIPTION
**Pull request by n-wbrown**
## Description

This PR adds a subclass of the roughing pump specific for the Ebara EV-A03-1 model of roughing pump. 

## Motivation and Context

The standard roughing pump model neglected a few important PVs. 

## How Has This Been Tested?

Standard tests and style checks succeed. 

## Where Has This Been Documented?

A docstring has been added specifically indicating the acceptable pump model No.

### CODE REVIEW

File: pcdsdevices/pump.py [hunk 1]
```diff
@@ -213,6 +213,13 @@ class PROPLC(Device):
     pump_at_speed = Cpt(EpicsSignalRO, ':AT_SPD_RBV', kind='normal')
 
 
+class Ebara_EV_A03_1(PROPLC):
+    """Class for the Ebara EV-A03-1 model of roughing pump"""
```
Comment (ZryletTC) [inline pcdsdevices/pump.py:217]:
```suggestion
    """Class for the Ebara EV-A03-1 model of roughing pump."""
```

File: pcdsdevices/pump.py [hunk 2]
```diff
@@ -213,6 +213,13 @@ class PROPLC(Device):
...
+    remote = Cpt(EpicsSignalWithRBV, ':REMOTE', kind='omitted')
+    alarm = Cpt(EpicsSignalRO, ':ALARM_OK_RBV', kind='omitted')
+    run_di = Cpt(EpicsSignalWithRBV, ':RUN_DI_RBV', kind='omitted')
```
Comment (ZryletTC) [inline pcdsdevices/pump.py:220]:
```suggestion
    run_di = Cpt(EpicsSignalRO, ':RUN_DI_RBV', kind='omitted')
```

**Comment by ZLLentz** [comment]
To clarify: I just want to make sure you have picked the correct `kind` designations for these components.

**Comment by n-wbrown** [comment]
@ZryletTC @ZLLentz Thanks for the comments! I think most of these channels fall into "expert user" territory but the `:ALARM_OK_RBV` should be straightforward enough be manageable in the normal screen.

**Comment by ZLLentz** [comment]
As long as you've carefully considered it, I'll take your conclusion as :+1: since I don't actually know anything about the device

**Comment by ZLLentz** [comment]
It looks like everything was taken care of and the tests pass. I am ready to merge this if you are done with it Nolan.

**Comment by n-wbrown** [comment]
Let's do it.

**Pull request merged**